CruCentralCoast/Misc/ImageManager.swift

```diff
@@ -15,16 +15,18 @@ class ImageManager {
 }
 
 extension ImageManager {
-    func fetch(_ urlString: String, _ completion: ((UIImage)->Void)? = nil) {
-        guard let url = URL(string: urlString) else {
-            print("WARN: Could not convert string to url: \(urlString).")
+    func fetch(_ urlString: String?, _ completion: ((UIImage)->Void)? = nil) {
```

Comment:
why change `urlString` to optional if you'll bail out when it's called with nil?

Reply:
By making it optional, it prevented an extra conditional check in each location that calls this method. Not sure which way is better

Reply:
my personal preference is to have the call sites manage passing in a non-nil object, then they need to know less about the inner working of the function and how it would handle a nil value, but obviously it'll perform the same either way so your call